sources/niLang/src/DataTablePath.h

```diff
@@ -46,8 +46,7 @@ enum eDataTablePathResultType
 };
 
 //! DataTable path operations.
-enum eDataTablePathOp
-{
+enum eDataTablePathOp {
```

Comment:
Please clear all the "indentation only" diffs. Those are only allowed in "Added clang-format" kind of diffs.